src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -389,11 +521,20 @@ def _process_epsilon(is_svc, args):
     as_rel_source = '_src'
 
     epsilon = args['epsilon']
-    if is_svc or not grouping_col or not eps_table:
+    # c code does SVR when epsilon is non-zero
+    if is_svc:
+        epsilon = 0.0
+        select_epsilon = '{epsilon}'.format(epsilon=epsilon)
+    elif not grouping_col or not eps_table:
         if eps_table:
-            plpy.warning('SVM: ignoring the input epsilon table!')
-        select_epsilon = str(epsilon)
+            plpy.warning('SVM warning: no grouping and '
+                         ' ignore the input epsilon table!')
+        # c code does SVC if epsilon is zero
+        if epsilon == 0: epsilon = 0.00001
```

Comment:
Also please look at the setting epsilon to low value part. I had added the `is_svc` boolean during the regression commit to avoid this.